### DISCUSSION TITLE
GDM Background...

### DISCUSSION THREAD

**Discussion by madcoda9000** [Anduin2017/AnduinOS#35] [Q&A] [answered]
Hello,

is there any way to change the gdm background? I don't like the grey background... :-)

Any help is welcome (PS: I've tried the gdm-settings app already, but that did not succeed..)

**Comment by Anduin2017** [comment]
Great question!! (Actually I also have no idea.. If someone can tell me how and I will be very glad to change the background in the prebuilt image for always grey is really not very nice.)

**Comment by Anduin2017** [comment]
I guess this might be working. Hasn't verify it yet..

https://askubuntu.com/questions/1227070/how-do-i-change-login-screen-theme-or-background-in-ubuntu-20-04

**Comment by Anduin2017** [comment, marked as answer]
I verified this:

https://github.com/PRATAP-KUMAR/gse-gdm-extension.git

On Anduin OS 1.2.0, and it works!

**Reply by madcoda9000** [reply]
Awesome! Yes this works!
